ZuluConverter.py

```diff
@@ -1,102 +1,44 @@
 from datetime import datetime
 from pytz import timezone
 
-
-    
-def convert_cst_to_zulu(cst_datetime):
-    # Define time zones
-    cst_tz = timezone('America/Chicago')  # Central Standard Time
-    zulu_tz = timezone('UTC')  # Coordinated Universal Time (Zulu)
-    date_format = "%Y-%m-%d %H:%M:%S"
-    cst = datetime.strptime(cst_datetime, date_format)
-    # Localize the input CST datetime
-    localized_cst = cst_tz.localize(cst)
-    
-    # Convert the localized CST datetime to Zulu time
-    zulu_datetime = localized_cst.astimezone(zulu_tz)
-    
-    zulu_datetime_formatted = zulu_datetime.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
-
-    return zulu_datetime_formatted
-
-
-
-
-#EST to Zulu
-def convert_est_to_zulu(est_datetime):
-    # Define time zones
-    est_tz = timezone('America/New_York')    # Eastern Standard Time
-    zulu_tz = timezone('UTC')   # Universal Time (Zulu)
-    date_format = "%Y-%m-%d %H:%M:%S"
-    est = datetime.strptime(est_datetime, date_format)
-    # Localize the input EST datetime
-    localized_est = est_tz.localize(est)
-
-    # Convert the localized EST datetime to Zulu time
-    zulu_datetime = localized_est.astimezone(zulu_tz)
-
-    zulu_datetime_formatted = zulu_datetime.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
-
-    return zulu_datetime_formatted
-
-
-#PST to Zulu
-def convert_pst_to_zulu(pst_datetime):
+#   myDateTime is user's input, zone is desired time zone (ex: CST, EST, PST, MST, HST, AKST)
+def time_converter(myDateTime, zone):
+    #Define time zones
+    zulu_tz = timezone('UTC')                   # Coordinated Universal Time (Zulu)

```

Comment:
shouldn't we only initialize these variables if needed in the `if` / `elif` blocks below?  only one of them will ever be needed, right?